### PR TITLE
Added remove chat member method.

### PR DESCRIPTION
**Pull request by AkJo**

### CODE REVIEW

File: api-reference/beta/resources/aaduserconversationmember.md [hunk 1]
```diff
@@ -13,26 +13,27 @@ Namespace: microsoft.graph
...
-|[Get channel member](../api/channel-get-members.md) | [conversationMember](conversationmember.md) collection | Get a member in a channel.|
 |[Create channel member](../api/channel-post-members.md) | [conversationMember](conversationmember.md) | Add a member to a channel. Only supported for `channel`with membershipType of `private`.|
+|[Get channel member](../api/channel-get-members.md) | [conversationMember](conversationmember.md) collection | Get a member in a channel.|
 |[Update channel member's role](../api/channel-update-members.md) | [conversationMember](conversationmember.md) | Update the properties of a member of the channel. Only supported for channel with membershipType of `private`.|
 |[Remove channel member](../api/channel-delete-members.md) | None | Delete a member from a channel. Only supported for `channelType` of `private`.|
 |[List chat members](../api/chat-list-members.md) | [conversationMember](conversationmember.md) collection | Get the list of all members in a chat.|
+|[Create chat member](../api/chat-post-members.md) | Location header | Add a member to a chat.|
```
Comment (Lauragra) [inline api-reference/beta/resources/aaduserconversationmember.md:34]: Should the return type here be conversationMember?

File: api-reference/beta/resources/aaduserconversationmember.md [hunk 2]
```diff
@@ -13,26 +13,27 @@ Namespace: microsoft.graph
...
+|[Get channel member](../api/channel-get-members.md) | [conversationMember](conversationmember.md) collection | Get a member in a channel.|
 |[Update channel member's role](../api/channel-update-members.md) | [conversationMember](conversationmember.md) | Update the properties of a member of the channel. Only supported for channel with membershipType of `private`.|
 |[Remove channel member](../api/channel-delete-members.md) | None | Delete a member from a channel. Only supported for `channelType` of `private`.|
 |[List chat members](../api/chat-list-members.md) | [conversationMember](conversationmember.md) collection | Get the list of all members in a chat.|
+|[Create chat member](../api/chat-post-members.md) | Location header | Add a member to a chat.| 
 |[Get chat member](../api/chat-get-members.md) | [conversationMember](conversationmember.md) | Get a member in a chat.|
-|[Add chat member](../api/chat-post-members.md) | Location header | Add a member to a chat.| 
+|[Delete chat member](../api/chat-post-members.md) | Location header | Remove a member from a chat.|
```
Comment (Lauragra) [inline api-reference/beta/resources/aaduserconversationmember.md:36]: This should have a return type of None.

Comment (Lauragra) [inline api-reference/beta/resources/aaduserconversationmember.md:36]: Let's make sure that the link titles in the Methods table match the topic titles. If we follow the guidance to use Create and Delete based on ContainsTarget =true, then we should rename the topic titles to be consistent with what we have in the table.

Comment (AkJo) [reply api-reference/beta/resources/aaduserconversationmember.md]: As discussed, I switched it back to Add/Remove.

File: api-reference/beta/resources/aaduserconversationmember.md (hunk 1 above)
Comment (AkJo) [reply api-reference/beta/resources/aaduserconversationmember.md]: We dont in this case. The API for add a chat member, returns an empty body with a location header that contains a path to the newly created member.

File: api-reference/beta/resources/aaduserconversationmember.md (hunk 2 above)
Comment (AkJo) [reply api-reference/beta/resources/aaduserconversationmember.md]: Fixed. The filepath was incorrect too.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**